_posts/2023-09-13-make-meaningful-contributions-at-osd.adoc

```diff
@@ -0,0 +1,9 @@
+---
+layout: post
+title: 'Vlog: Join us at Open Source Day on September 22nd, 2023 and Level Up Your Skills'
```

Comment:
title seems incorrect... should it be "Make Meaningful Contributions at Open Source Day" instead ?

Reply:
Oh, Sorry about that. Both of these should be fixed now! 
Thanks for pointing it out!